.travis.yml

```diff
@@ -1,17 +1,28 @@
-dist: bionic
+dist: focal
 language: php
+cache:
+  directories:
+  - $HOME/.composer
+  - $HOME/.npm
+env:
+  - SITEID=docstore
 
 php:
   - 8.0
```

Comment:
I think Jonathan wants to bump RW to PHP 8.1, so we *should* allow for a different PHP version at some stage.

We definitely do not want to wait for an image build before we lint, so maybe we can work round this by using the *builder* image to run the syntax checks on a mounted source directory. 

I think all our Drupals use a builder so that ought to be a safe choice.